app/views/posts/new.html.erb

```diff
@@ -0,0 +1,11 @@
+<h1>Create a new Post</h1>
+
+<%= form_with(model: @post, url: posts_path) do |form| %>
```

Comment:
- [ ] When I visit the new post route to create a post I get this error message. Kindly fix that. To check other functionalities I had to create a post from the rails console, so kindly fix this.

  ![image](https://github.com/Bellagirl-maker/blog_app/assets/54848363/015f7ec3-5889-4ef1-8169-8341ad144d71)

Reply:
@DammyShittu done with this change